h2o-core/src/main/java/water/rapids/FeatureImportance4BBM.java

```diff
@@ -0,0 +1,129 @@
+package water.rapids;
+
+import hex.AUC2;
+import hex.Model;
+//import hex.VarImp;
+import water.DKV;
+import water.Key;
+import water.Scope;
+import water.fvec.Frame;
+import water.fvec.Vec;
+import water.util.*;
+
+import java.util.*;
+
+
+public class FeatureImportance4BBM /* extends VarImp */ {
+    
+    private Model _model;
+    public Frame _ouput;
+    private Frame _train_fr; // original dataset used to train model
+    private Frame _validate_dataset; // original dataset used to train model
+    private Frame _predcited_frame;
+
+    private boolean _data_in_holdOut_form; // do I have the data split in train and validation, if not spliting will happen
+
+    private String _responce_col;
+
+    private AUC2 og_auc;
+
+    private double original_mse;
+    private double sq_err;
+            
+    public FeatureImportance4BBM(Model model) {_model = model;}
```

Comment:
I suggest changing the name to not contain a number. If you are implementing Permutation Feature importance, you can use this naming..

Reply:
Yes, good point. This was the initial name I put, will change it when refactoring